store/storetest/reaction_store.go

```diff
@@ -21,6 +22,8 @@ func TestReactionStore(t *testing.T, ss store.Store) {
 }
 
 func testReactionSave(t *testing.T, ss store.Store) {
+	assert := assert.New(t)
```

Comment:
We are not using this approach in other cases, we prefer to maintain consistency so I think is better if you pass the `t` in each assertion.

Reply:
You are right, I should have followed the convention. Apologies.